docs/tutorial.md

```diff
@@ -16,13 +16,60 @@ into online feature store for online feature serving.
 
 See [README](./../README.md#quickstart) for the instruction to run this demo.
 
+## Initialize FeatHub client
+
+```python
+client = FeathubClient(
+    config={
+        "processor": {
+            "processor_type": "flink",
```

Comment:
Do we intend to let the user start with Flink processor? It may be easier for users to start from the local processor. Otherwise, we may teach users to set up a standalone Flink cluster.

Reply:
Good point. It is fixed now.